src/deluge/processing/sound/sound.cpp

```diff
@@ -3833,6 +3854,7 @@ void Sound::writeToFile(bool savingSong, ParamManager* paramManager, Arpeggiator
 		storageManager.writeOpeningTagBeginning("arpeggiator");
 		storageManager.writeAttribute("mode", arpModeToString(arpSettings->mode));
 		storageManager.writeAttribute("numOctaves", arpSettings->numOctaves);
+		storageManager.writeAttribute("numRatchets", arpSettings->numRatchets);
```

Comment:
not used by sounds - sounds are using the unpatched param instead

Reply:
this is for the number of ratchets, which is a "menu" parameter. It is similar to octaves and cannot be automated